cmd/ef_tests/types.rs

```diff
@@ -177,12 +177,11 @@ pub struct Transaction {
     pub blob_versioned_hashes: Option<Vec<H256>>,
     pub hash: Option<H256>,
     pub sender: Address,
-    #[serde(deserialize_with = "crate::serde_utils::h160::deser_hex_str")]
+    #[serde(deserialize_with = "ethereum_rust_core::serde_utils::h160::deser_hex_str")]
     pub to: Address,
```

Comment:
We could change the type of `to`  to `TxKind` as it already has `Deserialize` implemented